GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -24,6 +26,14 @@ public MacPlatform()
         public override GVFSPlatformConstants Constants { get; } = new MacPlatformConstants();
         public override IPlatformFileSystem FileSystem { get; } = new MacFileSystem();
 
+        public override string GVFSConfigPath
+        {
+            get
+            {
+                return Path.Combine(this.Constants.GVFSBinDirectoryPath, LocalGVFSConfig.FileName);

```

Comment:
Note: I am not an expert on how file permissions work on Mac, so there might be no issues here!

Does `sudo` care at all about what group a user is in?  I'm wondering if two users are modifying `/usr/local/vfsforgit/gvfs.config` is there any risk that one of them will block the other from being able to make further changes (using `sudo`)?

Similarly, is there any chance that one user's changes might prevent another users from being able to read the file (without using `sudo`)?

Reply:
Great question(s)!

Root is its own account, so when running a command via sudo, `root` will be the owner of this file. The mode bits on the file will be such that `root` is the only account that can modify the file, while it will be readable by everyone. If two different users attempt to modify this file via a `sudo` command, it will be the `root` account that is making changes.

/cc @jeffhostetler to check my understanding :)